osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -604,12 +605,46 @@ - (IBAction)discardPreferences:(id)sender {
     [self discardPreferences];
 }
 
+- (IBAction)kaliteUninstall:(id)sender {
+        
+    // Get the KA Lite application directory path.
+    NSString *appPath = [[NSBundle mainBundle] bundlePath];
+    NSRange searchLastWord = [appPath rangeOfString:@"/" options:NSBackwardsSearch];
```

Comment:
Is there any other way to get the file path for the uninstall tool without doing these?